client-course-schedulizer/src/components/AddSectionButton/AddSectionButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconButton, Popover } from "@material-ui/core";
+import { Add } from "@material-ui/icons";
+import React, { MouseEvent, useState } from "react";
+import { AddSectionPopover } from "../AddSectionPopover";
+import "./AddSectionButton.scss";
+
+export const AddSectionButton = () => {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? "simple-popover" : undefined;
```

Comment:
Either we should move this logic to be it's own Hook for future popover use or we can this package to manage the popover state. https://www.npmjs.com/package/material-ui-popup-state